src-input/duk_bi_object.c

```diff
@@ -293,9 +300,16 @@ DUK_INTERNAL duk_ret_t duk_bi_object_constructor_is_sealed_frozen_shared(duk_con
 		                          1 :               /* lightfunc always frozen and sealed */
 		                          (is_frozen ^ 1)); /* buffer sealed but not frozen (index props writable) */
 	} else {
-		h = duk_require_hobject(ctx, 0);
-		rc = duk_hobject_object_is_sealed_frozen_helper((duk_hthread *) ctx, h, is_frozen /*is_frozen*/);
-		duk_push_boolean(ctx, rc);
+		h = duk_get_hobject(ctx, 0);
+		if (h != NULL) {
+			rc = duk_hobject_object_is_sealed_frozen_helper((duk_hthread *) ctx, h, is_frozen /*is_frozen*/);
+			duk_push_boolean(ctx, rc);
```

Comment:
Footprint-wise better would be:

    duk_push_boolean(ctx, h == NULL ? 1 : duk_hobject_object_is_sealed_frozen_helper(...));